composer.json

```diff
@@ -101,6 +101,7 @@
         "drupal/oembed_providers": "^1.1",
         "drupal/entity_clone": "^1.0@beta",
         "drupal/back_to_top": "^2.0"
+        "drupal/environment_indicator": "^4.0"
```

Comment:
This line looks weird, not sure why the colon has a background.

Reply:
Updating PR - there was a missing comma on the line above.